lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
       <nav class="page-content-nav">
         <a href="" class="nav-item is-active">My Teams</a>
-        <a href="" class="nav-item">Browse All Teams</a>
+        <a href="" class="nav-item">Browse</a>
       </nav>
 
       <div class="page-content-main">
-        <p>Blah blah blah. Go Team!</p>
+
+        <div class="search-tools">
+          <span class="search-count">Viewing 12 of 24 topics</span>
```

Comment:
[My feedback about the naming of the parent element above](https://github.com/edx/edx-platform/pull/8468/files#r33264867) would affect the ``search-`` prefix of these elements as well.